SU2_CFD/src/solvers/CSolver.cpp

```diff
@@ -3316,29 +3316,23 @@ void CSolver::LoadInletProfile(CGeometry **geometry,
   string Interpolation_Function, Interpolation_Type;
   bool Interpolate = false;
 
-  su2double *Normal = new su2double[nDim];
-
   unsigned long Marker_Counter = 0;
 
-  bool turbulent = (config->GetKind_Solver() == RANS ||
-                    config->GetKind_Solver() == INC_RANS ||
-                    config->GetKind_Solver() == ADJ_RANS ||
-                    config->GetKind_Solver() == DISC_ADJ_RANS ||
-                    config->GetKind_Solver() == DISC_ADJ_INC_RANS);
-
   unsigned short nVar_Turb = 0;
-  if (turbulent)
-    switch (config->GetKind_Turb_Model()) {
-      case SA: case SA_NEG: case SA_E: case SA_COMP: case SA_E_COMP:
-        nVar_Turb = 1;
-        break;
-      case SST: case SST_SUST:
-        nVar_Turb = 2;
-        break;
-      default:
-        SU2_MPI::Error("Specified turbulence model unavailable or none selected", CURRENT_FUNCTION);
-        break;
-    }
+
+  switch (config->GetKind_Turb_Model()) {
+    case NONE:
+      break;
+    case SA: case SA_NEG: case SA_E: case SA_COMP: case SA_E_COMP:
+      nVar_Turb = 1;
+      break;
+    case SST: case SST_SUST:
+      nVar_Turb = 2;
+      break;
+    default:
+      SU2_MPI::Error("Specified turbulence model unavailable or none selected", CURRENT_FUNCTION);
```

Comment:
"none selected" is covered by the switch.

Reply:
Not sure what you mean, we still need to cover the case when there is no turbulence solver.
I cleaned it up more to avoid the switch.